python_modules/dagster/setup.py

```diff
@@ -28,6 +28,10 @@ def get_version() -> str:
 # grpcio 1.44.0 is the min version compatible with both protobuf 3 and 4
 GRPC_VERSION_FLOOR = "1.44.0"
 
+# Pinned due to hangs that appear to have been introduced in later versions of grpcio:
+# https://github.com/grpc/grpc/issues/36256
+GRPC_VERSION_CEILING_EXCLUSIVE = "1.60.0"
```

Comment:
Can we link to more supporting evidence? Even just backlink to this PR and then add any relevant context to PR (screenshots from slack etc)